Externals/curl/curl.vcxproj

```diff
@@ -358,7 +358,9 @@
     <ClInclude Include="lib\x509asn1.h" />
   </ItemGroup>
   <ItemGroup>
-    <ResourceCompile Include="lib\libcurl.rc" />
+    <ResourceCompile Include="lib\libcurl.rc">
```

Comment:
Indeed, that should also work. There are a few other files that are also marked as excluded, though, which were added in 3b3551f04092fb96d67274a9f83ba1f509e9dcef (by you). Any reason why that was done, instead of deleting the relevant lines?

Reply:
I'll just delete the item group instead, then.